tools/mpremote/mpremote/commands.py

```diff
@@ -334,6 +334,50 @@ def do_filesystem_recursive_rm(state, path, args):
             print(f"removed: '{path}'")
 
 
+def human_size(size, decimals=1):
+    for unit in ['B', 'K', 'M', 'G', 'T']:
+        if size < 1024.0 or unit == 'T':
+            break
+        size /= 1024.0
+    return f"{size:.{decimals}f}{unit}" if unit != 'B' else f"{int(size)} {unit}"
```

Comment:
Did you intend to add the unit `B` in the case of bytes?  I think it can be left out, no unit means bytes.

Reply:
No the intent was indeed to drop it.